loader/loader.go

```diff
@@ -1274,8 +1275,12 @@ func (l *Loader) getMydumpMetadata() error {
 	metafile := filepath.Join(l.cfg.LoaderConfig.Dir, "metadata")
 	loc, _, err := dumpling.ParseMetaData(metafile, l.cfg.Flavor)
 	if err != nil {
+		toPrint, err := ioutil.ReadFile(metafile)
```

Comment:
Do we need to keep the original `err` returned from `dumpling.ParseMetaData`? if so, ditto for another call to `dumpling.ParseMetaData`.

Reply:
https://github.com/pingcap/dm/pull/988/commits/4f4be9b30ede7f88459670a4de519ff660fab640